post_processing/plot_gpu_scaling_utils.jl

```diff
@@ -0,0 +1,36 @@
+using JLD2
+function get_jld2data(output_dir, job_id, s)
+    FT = Float64
+    nprocs_clima_atmos = Int[]
+    ncols_per_process = Int[]
+    walltime_clima_atmos = FT[]
+    found = false
+    for foldername in readdir(output_dir)
+        if occursin(job_id, foldername) && occursin(s, foldername)
+            nprocs_string = split(split(foldername, s)[end], "process")[1]
+            file = joinpath(
+                output_dir,
+                foldername,
+                "scaling_data_$(nprocs_string)_processes.jld2",
+            )
```

Comment:
```suggestion
            file = joinpath(
                output_dir,
                foldername,
                "output_active",
                "scaling_data_$(nprocs_string)_processes.jld2",
            )
```

I think this change will fix the new path from the OutputPathGenerator, see [here](https://buildkite.com/clima/climaatmos-target-gpu-simulations/builds/258#018ebf56-f2bc-47f5-b611-a38ee78ac817).

Reply:
Perfect, thanks for launching that